### ISSUE TITLE
When opening extension options, do not use any currently open tabs

### ISSUE THREAD

**Issue by DanAtkinson** [deanoemcke/thegreatsuspender#239]
I think this is possibly a Chromium 'bug' as well but, if I open the extension options, it will navigate to the first suspended tab and open in that - presumably because the url has been changed to something else.

This isn't so bad in itself, but coupled with the fact that, when you press the 'save' button and the window closes, this causes something of an annoyance for the user.

I'm not sure if you have control over this, but ideally, the options should not open in an existing tab (unless an options tab is already open) but should instead open a new tab.

**Comment by deanoemcke** [comment]
This is a duplicate of issue #222.
Unable to work out why this happens - but it only happens from the right-click menu, not from the popup menu. If it is indeed a chromium bug then it will be outside of my ability to fix.
